velox/docs/functions/spark/string.rst

```diff
@@ -141,6 +141,12 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT overlay('Spark SQL', 'tructured', 2, 4); -- "Structured SQL"
         SELECT overlay('Spark SQL', '_', -6, 3); -- "_Sql"
 
+.. spark:function:: string_repeat(str, n) -> varchar
```

Comment:
string_repeat -> repeat
And then document "string_repeat" separately to just indicate it's an alias for "repeat" function.